stob/domain.h

```diff
@@ -196,6 +203,13 @@ M0_INTERNAL int m0_stob_domain_destroy(struct m0_stob_domain *dom);
  */
 M0_INTERNAL int m0_stob_domain_destroy_location(const char *location);
 
+/**
+ * Truncates a stob domain.
+ *
+ * It will delete all the record present in stob domain.
+ */
+M0_INTERNAL int m0_stob_domain_truncate_location(const char *location);
```

Comment:
Why do we need m0_stob_domain_truncate_location()? I do not see anyone other than m0_stob_domain_truncate() making call to this function.

Reply:
okay I will remove this function and make appropriate changes